scripts/generate-redirects-from-migrations/generate-redirects.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	fmt.Println("This script creates redirect entries for migrated projects.")
+	fmt.Print("You will be prompted for access tokens for Renku and GitLab, respectively.\n\n")
```

Comment:
```suggestion
	fmt.Print("You will be prompted for access tokens for Renku and GitLab, respectively. Note that both tokens need to have admin rights.\n\n")
```

Reply:
Please also add the text that explains that for renku we need the `_renku_session` cookie. It should be in the text that prompts for it, not just in the readme. Same for gitlab it should be clear which token should be used.

Reply:
Clarified in 7e2eb71f6cd5df8fffd4e9e2905722a20dd91bb4